src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -109,6 +109,13 @@ export default class Result extends React.Component {
     return <Note>{t(`Displaying up to ${NUMBER_OF_SERIES_BY_DAY} results`)}</Note>;
   }
 
+  truncate(string) {
+    if (string.length > 60) {
```

Comment:
Let's make this a constant, also we truncate tooltips with a different value, should they be the same?

Reply:
Yes good point, thank you!